packages/terra-hookshot/src/_HookshotUtils.js

```diff
@@ -0,0 +1,420 @@
+const BOUNDS_FORMAT = ['left', 'top', 'right', 'bottom'];
+const MIRROR_LR = {
+  center: 'center',
+  left: 'right',
+  right: 'left',
+};
+
+const MIRROR_TB = {
+  middle: 'middle',
+  top: 'bottom',
+  bottom: 'top',
+};
+
+// Same as native getBoundingClientRect, except it takes into account parent <frame> offsets
```

Comment:
we should document each of these functions to explain what they do and what parameters are passed in